pkg/crc/preflight/preflight_darwin.go

```diff
@@ -106,6 +106,18 @@ var resolverPreflightChecks = []Check{
 	},
 }
 
+var daemonLaunchdChecks = []Check{
+	{
+		configKeySuffix:    "check-daemon-launchd-plist",
+		checkDescription:   "Checking if crc daemon plist file present and loaded",
+		check:              checkIfDaemonPlistFileExists,
+		fixDescription:     "Adding crc daemon plist file and loading it",
+		fix:                fixDaemonPlistFileExists,
+		cleanupDescription: "Unloading and removing the daemom plist file",
+		cleanup:            removeDaemonPlistFile,
```

Comment:
Not sure we should be emphasizing the plist file in these messages. What matters is that we are setting up launchd to start the daemon with the session?

Reply:
@cfergeau yes we are using launchd to start the daemon but plist is driving force something like in linux the unit file for systemd operations.